adapter.go

```diff
@@ -61,6 +61,19 @@ func NewAdapter(url string) persist.Adapter {
 	return a
 }
 
+// Can use provided database name.
+func NewDBAdapter(url, name string) persist.Adapter {
```

Comment:
Please do not create another constructor method. Just do it as how we did it in Xorm Adapter: https://github.com/casbin/xorm-adapter/blob/6d86407ce2631ca1f6f6943374c6129c5b03e190/adapter.go#L50-L75